packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import isEmpty from 'lodash/isEmpty';
+
+import { Button } from '../../../primitives/Button';
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { IconClose } from '../../../primitives/Icon/icons/IconClose';
+import { Image } from '../../../primitives/Image';
+import { Text } from '../../../primitives/Text';
+import { Link } from '../../../primitives/Link';
+import { useThemeBreakpoint } from '../../../hooks/useThemeBreakpoint';
+
+export default function BannerMessage({
+  alignment = 'right',
+  ...props
+}): JSX.Element | null {
+  const {
+    body,
+    header,
+    image,
+    onClose,
+    position,
+    primaryButton,
+    secondaryButton,
+  } = props;
+
+  const hasPrimaryButton = !isEmpty(primaryButton);
+  const hasSecondaryButton = !isEmpty(secondaryButton);
+  const breakpoint = useThemeBreakpoint();
+  console.log(typeof breakpoint);
+
+  return (
+    <Flex
+      className={`amplify-in-app-messaging-banner__container amplify-in-app-messaging-banner-${position} ${String(
+        breakpoint
+      )}`}
+    >
+      <Flex className="amplify-in-app-messaging-banner__content-container">
+        {image?.src ? (
+          <Flex className="amplify-in-app-messaging-banner__image-container">
+            <Image
+              src={image.src}
+              className="amplify-in-app-messaging-banner__image"
+              alt="Message Image"
+            />
+          </Flex>
+        ) : null}
+        <Flex className="amplify-in-app-messaging-banner__text-container">
+          {header?.content ? <Heading>{header.content}</Heading> : null}
+          {body?.content ? (
+            <Text className="amplify-in-app-messaging-banner__body">
+              {body.content}
+            </Text>
+          ) : null}
+        </Flex>
+        <Link>
+          <IconClose ariaLabel="Close" onClick={onClose} />
+        </Link>
```

Comment:
We can replace this with the `CloseIconButton` ([merged here](https://github.com/aws-amplify/amplify-ui/pull/2342))